app/uk/gov/hmrc/gform/formtemplate/FormComponentMaker.scala

```diff
@@ -89,6 +89,7 @@ class FormComponentMaker(json: JsValue) {
     toOpt((json \ "validators").validateOpt[List[FormComponentValidator]], "/validators").map(_.toList.flatten)
   lazy val mandatory: Option[String] = (json \ "mandatory").asOpt[String]
   lazy val multiline: Option[String] = (json \ "multiline").asOpt[String]
+  lazy val rows: Option[Int] = (json \ "rows").asOpt[Int]
```

Comment:
I believe there exists some reasonable default here, can we make `rows` an `Int` instead of an `Option[Int]`?

Or let's wait on frontend implementation instead, to see how well this plays with it.

Reply:
In a library **play-frontend**, component **Textarea** **rows** has been defaulted to 5. As per discussion with Alex, if we don't provide rows in a template then we shouldn't provide any value of rows to library.

Reply:
Fair enough

Reply:
But actually I'm still interested, how is this `Option` beneficial? How are you avoid calling `.getOrElse` on it to provide `Int` value to the **play-frontend** ?

Reply:
Will use **fold** on **rows: Option[Int]** with default value without row parameter.

Reply:
I prefer making it a Int instead of Optional, especially because we have a way to add  the default value to JSON if it doesnt exist